lib/renderer/api/remote.js

```diff
@@ -304,6 +304,10 @@ ipcRenderer.on('ELECTRON_RENDERER_RELEASE_CALLBACK', function (event, id) {
   callbacksRegistry.remove(id)
 })
 
+process.on('exit', () => {
+  ipcRenderer.sendSync('ELECTRON_BROWSER_CONTEXT_RELEASE')
```

Comment:
Just to confirm, this is only needed for sanboxed renderer processes right? Or was this issue also affecting non-sandboxed renderers?